README.md

```diff
@@ -1506,11 +1506,12 @@ Thanks for the course and good luck to students!
 
 ---
 
-# Hello to everyone! :smiley_cat:
-My name is [Olga](https://github.com/ORybka) and I am very glad to join this community :grin:
-**Good luck to everyone!**
-And meet my cat who saw some things in this world :D
+# Yo, hello, kottans
```

Comment:
Please get the latest changes from original repo and fix conflicts

Reply:
so, I need to return line 1510-1512 to readme?